addons/web/static/tests/views/fields/ace_editor_field_tests.js

```diff
@@ -259,4 +259,36 @@ QUnit.module("Fields", (hooks) => {
         await click(target, ".o_form_button_save");
         assert.verifySteps(['web_save: [[1],{"foo":"a"}]']);
     });
+
+    QUnit.test("Save and Discard buttons will become invisible after saving", async (assert) => {
+        serverData.models.partner.records.forEach((rec) => {
+            rec.foo = false;
+        });
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            resId: 1,
+            serverData,
+            arch: `<form>
+                <field name="display_name" />
+                <field name="foo" widget="code" />
+            </form>`,
+            mockRPC(route, args) {
+                if (args.method) {
+                    assert.step(`${args.method}: ${JSON.stringify(args.args)}`);
```

Comment:
I don't think you need to test the rpcs in this test (already done in the previous one)